modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -184,17 +184,16 @@ CAUTION: This feature is marked xref:project-docs:compatibility.adoc#interface-s
 Expect a promotion to _Committed_ API in a future release.
 
 The code snippet below shows how to run a simple query to fetch 10 random rows from travel-sample and print the results, 
-the assumption is that the `airline` collection exists within a scope `us`.
+the assumption is that the `airline` collection exists within a scope `myScope`.
 
 [source,golang]
 ----
-myCollection := "airline"
-query := fmt.Sprintf("SELECT x.* FROM %s x LIMIT 10;", myCollection)
-        rows, err := us.Query(query, &gocb.QueryOptions{})
-        // check query was successful
-        if err != nil {
-                panic(err)
-        }
+myScope := cluster.Bucket("mybucket").Scope("myscope")
```

Comment:
It would be good if this pointed to an actual scope in `travel-sample` (i.e 'inventory'), so that we can run the example code, but can be done at a later time. 👍